bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBaseThingHandler.java

```diff
@@ -178,11 +233,48 @@ protected void cacheStateType(String stateName, int type) {
         }
     }
 
+    private void updateUnits(List<SomfyTahomaState> attributes) {
+        for (SomfyTahomaState attr : attributes) {
+            if ("core:MeasuredValueType".equals(attr.getName()) && attr.getType() == TYPE_STRING) {
+                switch ((String) attr.getValue()) {
+                    case "core:TemperatureInCelcius":
```

Comment:
Missing case for case "core:TemperatureInCelsius" (with s in Celsius)

Reply:
Added.
I can confirm that with the Cozytouch I find a value with a c.